sktime/transformations/panel/dictionary_based/_sfa.py

```diff
@@ -115,6 +115,8 @@ class SFA(BaseTransformer):
         "X_inner_mtype": "numpy3D",  # which mtypes do _fit/_predict support for X?
         "y_inner_mtype": "pd_Series_Table",  # which mtypes does y require?
         "requires_y": True,  # does y need to be passed in fit?
+        "python_dependencies": ["numba", "joblib"],
+        "tests:skip_all": True #added entry test:skip tag
```

Comment:
you shouldn't have to modify the `_sfa.py` file anymore, right? since you're no longer working on SFA, but focusing on `compose_distance.py`